epoxy-processor/src/main/java/com/airbnb/epoxy/EpoxyProcessor.java

```diff
@@ -385,6 +390,10 @@ private MethodSpec buildHashCode(ClassToGenerateInfo helperClass) {
     }
 
     for (AttributeInfo attributeInfo : helperClass.getAttributeInfo()) {
+      if (!attributeInfo.useInHash()) {
```

Comment:
You should also include this check in line 386 since the temp variable won't be needed if that attribute isn't included in the hash